agent/agent.go

```diff
@@ -513,12 +514,26 @@ func (a *Agent) killProcs(tc *taskContext, ignoreTaskGroupCheck bool) {
 				grip.Critical(msg)
 			}
 		}
-
 		if tc.logger != nil && !tc.logger.Closed() {
 			tc.logger.Task().Infof("cleaned up processes for task: %s", tc.task.ID)
 		} else {
 			grip.Infof("cleaned up processes for task: %s", tc.task.ID)
 		}
+
+		if tc.logger != nil && !tc.logger.Closed() {
+			tc.logger.Task().Info("cleaning up Docker artifacts")
+		} else {
+			grip.Info("cleaning up Docker artifacts")
+		}
+		if err := docker.Cleanup(ctx, tc.logger.Task()); err != nil {
```

Comment:
It might make sense to add a timeout to this context, in case cleaning up Docker hangs for reason. I'm concerned we might strand hosts.

Reply:
done.